release.nix

```diff
@@ -57,7 +71,12 @@ in allJobs // {
       meta.description = "All jobs required to pass CI";
       constituents =
           collect isDerivation allJobs.R1903.native
-       ++ collect isDerivation allJobs.R1909.native;
+       ++ collect isDerivation allJobs.R1909.native
+       # Windows cross compiling with GHC 8.8.2 is broken
```

Comment:
Do you remember *why* it was broken? Guess we should ideally leave a comment (can also be a hydra link).

I do think we should make sure cross-windows *does* work. We use that in production...

Reply:
```
/nix/store/fdl2b72bmvrjbn1y690fj1901805zpjv-x86_64-pc-mingw32-binutils-2.31.1/bin/x86_64-pc-mingw32-ld: libraries/base/dist-install/build/HSbase-4.13.0.0.p_o: too many sections (55036)
/nix/store/fdl2b72bmvrjbn1y690fj1901805zpjv-x86_64-pc-mingw32-binutils-2.31.1/bin/x86_64-pc-mingw32-ld: final link failed: file too big
```

https://hydra.iohk.io/build/1676575/nixlog/12

Reply:
turning off split sections might fix that, at the cost of link time gc

Reply:
Turning off split sections seemed to fix it, thanks.